airbyte-integrations/connectors/source-smartsheets/source_smartsheets/spec.json

```diff
@@ -82,6 +82,31 @@
         "default": "2020-01-01T00:00:00+00:00",
         "order": 2,
         "airbyte_hidden": true
+      },
+      "metadata_fields": {
+        "title": "Metadata Fields",
+        "type": "array",
+        "examples": [
+          "sheetcreatedAt",
+          "sheetid",
+          "sheetmodifiedAt",
+          "sheetname",
+          "sheetpermalink",
+          "sheetversion",
+          "sheetaccess_level",
+          "row_id",
+          "row_access_level",
+          "row_created_at",
+          "row_created_by",
+          "row_expanded",
+          "row_modified_by",
+          "row_parent_id",
+          "row_permalink",
+          "row_number",
+          "row_version"
```

Comment:
What happen when I use add a field which is not valid? It's going to break? Can we have the complete list of metadata fields?

Reply:
There is a list provided in the updated `docs/integrations/sources/smartsheets.md`. In its current state, it allows me to input an incorrect metadata field, says the connection tests fine, but fails to sync.

Reply:
The best here is to change the param format to be an enum where the user can select from the actual list.

Reply:
Thanks, I'll have some time to take a look and make some changes to it next week.